presto-main/src/test/java/io/prestosql/operator/TestAnnotationEngineForScalars.java

```diff
@@ -465,6 +465,7 @@ public void testSimpleInjectionScalarParse()
     @Description("Parametric scalar with type injected though constructor")
     public static class ConstructorInjectionScalarFunction
     {
+        @SuppressWarnings("unused")
         private final Type type;
```

Comment:
This field would be parsed during tests.

Reply:
i removed the field and the `TestAnnotationEngineForScalars` test class still passes.
Are we missing an assertion somewhere?

Reply:
Thanks for your review. It seems that I misunderstand the code, the ctor argument with annotation would be parsed instead of the field. The unused field is able to be removed.